docs/code/StatisticsBackendTests.cpp

```diff
@@ -319,6 +319,22 @@ int get_graph_examples(
     return 0;
 }
 
+void dump_load_examples()
+{
+    {
+        //CONF-DUMP-LOAD-EXAMPLE
+        // Save the database to a file
+        StatisticsBackend::dump_database("new_backend.dump");
+
+        // Reset the Backend to empty the current database contents
+        StatisticsBackend::reset();
+
+        // Load an old backup to the emptied Backend
+        StatisticsBackend::load_database("old_backend.dump");
```

Comment:
I suggest rename `.dump` to `.json.` I think is better for the user to visualize quickly that the format is a known standard.
```suggestion
        StatisticsBackend::dump_database("new_backend.json");

        // Reset the Backend to empty the current database contents
        StatisticsBackend::reset();

        // Load an old backup to the emptied Backend
        StatisticsBackend::load_database("old_backend.json");
```